kumascript/macros/SpecData.json

```diff
@@ -1669,6 +1669,11 @@
     "url": "https://w3c.github.io/web-nfc/",
     "status": "Draft"
   },
+  "WebOTP": {
+    "name": "WebOTP",
+    "url": "https://w3c.github.io/web-otp/",
```

Comment:
I get a 404 with this URL. Should this be https://wicg.github.io/web-otp/ instead?

Reply:
ugh sorry yes - thanks for catching!